src/stash/protos/objects.proto

```diff
@@ -458,7 +461,7 @@ message AuditLogEventV1 {
 
     message CreateClusterReplicaV1 {
         string cluster_id = 1;
-        string cluser_name = 2;
+        string cluster_name = 2;
```

Comment:
:100:

Reply:
Is it safe to change protobuf field names without bumping the number?

Reply:
Yes, names and orders of fields can change without affecting backwards-compatibility. The important thing is that the tag value remains unchanged.